recipe/build-libadios.sh

```diff
@@ -53,31 +53,37 @@ cmake               \
     -S "${SRC_DIR}" \
     -B build        \
     -GNinja         \
-    -DCMAKE_BUILD_TYPE=Release                \
-    -DBUILD_SHARED_LIBS=ON                    \
-    -DBUILD_TESTING=${RUN_TESTS}              \
+    -DADIOS2_BUILD_EXAMPLES=OFF               \
+    -DADIOS2_HAVE_ZFP_CUDA=OFF                \
+    -DADIOS2_INSTALL_GENERATE_CONFIG=OFF      \
+    -DADIOS2_RUN_INSTALL_TEST=OFF             \
+    -DADIOS2_RUN_INSTALL_TEST=ON              \
     -DADIOS2_USE_BZip2=ON                     \
     -DADIOS2_USE_HDF5=ON                      \
     -DADIOS2_USE_MPI=${USE_MPI}               \
     -DADIOS2_USE_PNG=ON                       \
     -DADIOS2_USE_Python=ON                    \
     -DADIOS2_USE_ZeroMQ=ON                    \
     -DADIOS2_USE_ZFP=ON                       \
-    -DADIOS2_HAVE_ZFP_CUDA=OFF                \
-    -DADIOS2_BUILD_EXAMPLES=OFF               \
-    -DADIOS2_RUN_INSTALL_TEST=OFF             \
-    -DPython_EXECUTABLE:FILEPATH="${PYTHON}"  \
-    -DPython_INCLUDE_DIR="$(${PYTHON} -c "from sysconfig import get_paths as gp; print(gp()['include'])")" \
+    -DBUILD_SHARED_LIBS=ON                    \
+    -DBUILD_TESTING=${RUN_TESTS}              \
+    -DCMAKE_BUILD_TYPE=Release                \
     -DCMAKE_INSTALL_LIBDIR=lib                \
     -DCMAKE_INSTALL_PREFIX="${PREFIX}"        \
+    -DPython_EXECUTABLE:FILEPATH="${PYTHON}"  \
+    -DPython_INCLUDE_DIR="$(${PYTHON} -c "from sysconfig import get_paths as gp; print(gp()['include'])")" \
     -DPNG_PNG_INCLUDE_DIR="${PREFIX}"
 
-cmake --build build "-j${CPU_COUNT}" -v
+cmake --build build "-j$((CPU_COUNT*2))" -v
```

Comment:
There is some io components in these builds, having more process than cores would offset this.

Reply:
The problem is that we will run into random system process kills if we oversubscribe the runners. Please do not use more processes * threads than 2 (3 for macOS runners) to avoid.

Reply:
I see.